bin/edgeR_miRBase.r

```diff
@@ -79,7 +79,7 @@ for (i in 1:2) {
     }
 
     # Make MDS plot (only perform with 3 or more samples)
-    if (length(filelist[[1]]) > 2){
+    if (length(ncol(dataNorm$counts)) > 2){
```

Comment:
@atrigila ncol(dataNorm$counts) returns a single integer value representing the number of columns, wouldn't wrapping it with length() always return 1? Making this condition ineffective. Shouldn't we just check ncol(dataNorm$counts) > 2 directly instead?